charts/opentelemetry-collector/values.schema.json

```diff
@@ -551,6 +551,24 @@
     "hostNetwork": {
       "type": "boolean"
     },
+    "hostAliases": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "hostnames": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "ip": {
+            "type": "string"
+          }
```

Comment:
```suggestion
```

Normally when we are planning to use the object directly in the chart we don't do our property enforcement.  This lets the feature scale and kubernetes changes.

Reply:
Hi @TylerHelmuth,
Problem fixed, regards.